packages/webcomponents/src/ui-components/form/iframe-input.tsx

```diff
@@ -0,0 +1,110 @@
+import { Component, h, Method, Prop, State } from "@stencil/core";
+import { FrameCommunicationService } from "../../utils/frame-comunication-service";
+import { StyledHost } from "../styled-host/styled-host";
+import { MessageEventType } from "../../components/payment-method-form/message-event-types";
+import { FormControlErrorText } from "./form-helpers/form-control-error-text";
+import packageJson from '../../../package.json';
+
+@Component({
+  tag: "iframe-input",
+  shadow: true,
```

Comment:
Can we have this inherit styles from `Checkout` or `PaymentForm`'s `StyledHost` instead?